File: katas/7 kyu/binary-addition/solution.js
// binary-addition
/* ------------------------------- MY SOLUTION ------------------------------ */
function addBinary(a,b) {
  return (a+b).toString(2)
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
const chai = require("chai");
const Test = chai.assert.strictEqual; // replace Test.assertEqual with Test
chai.config.truncateThreshold=0;

// const Test = require('@codewars/test-compat');

describe("addBinary(1,2)", function() {
  var results1 = addBinary(1,2);
  it("Should return \"11\"", function() {
    Test(results1, "11");
  });
});

File: katas/7 kyu/binary-addition/solution.py
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def add_binary(a,b):
  return bin(a+b)[2:]
# ----------------------------------- TEST ----------------------------------- #
@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(add_binary(1,1),"10")
        test.assert_equals(add_binary(0,1),"1")
        test.assert_equals(add_binary(1,0),"1")
        test.assert_equals(add_binary(2,2),"100")
        test.assert_equals(add_binary(51,12),"111111")